_includes/participation.html

```diff
@@ -0,0 +1,6 @@
+
+<div>
+    <h4 style="text-align: center;">Participation Credit</h4>
+    <h6 class="footer-text">The CSG will sponsor a student for class particpation if the student attends 2 lectures, participates in a competition, or becomes an officer.</h6>
```

Comment:
This text is very small style wise. Why are you using h6?
I still think it should be it's own page, but if you can get someone else to approve it go ahead.